src/Item.h

```diff
@@ -239,7 +239,7 @@ class cItems  // tolua_export
 
 	void    Add   (short a_ItemType, char a_ItemCount, short a_ItemDamage)
 	{
-		push_back(cItem(a_ItemType, a_ItemCount, a_ItemDamage));
+		emplace_back(cItem(a_ItemType, a_ItemCount, a_ItemDamage));
```

Comment:
Might as well drop the cItem() since you're constructing it in-place with... emplace.